src/main/kotlin/com/openlattice/chronicle/study/StudyUpdate.kt

```diff
@@ -0,0 +1,32 @@
+package com.openlattice.chronicle.study
+
+import com.google.common.base.Preconditions
+import java.time.OffsetDateTime
+
+/**
+ * @author Solomon Tang <solomon@openlattice.com>
+ */
+data class StudyUpdate(
+    val title: String?,
+    val description: String?,
+    val startedAt: OffsetDateTime?,
+    val endedAt: OffsetDateTime?,
+    val lat: Double?,
+    val lon: Double?,
+    val group: String?,
+    val version: String?,
+    val settings: Map<String, Any>?,
+    val contact: String?,
+) {
+    init {
+        Preconditions.checkArgument(
+            title == null || title.isNotEmpty(),
+            "Title cannot be blank."
+        )
+        Preconditions.checkArgument(
+            contact == null || contact.isNotEmpty(),
+            "Contact cannot be blank."
+        )
+    }
```

Comment:
What about all the other fields? I'd say it's better to omit them if you're not going to check them.

Reply:
I only want to enforce fields that are required to never be blank. optional fields should be able to remain or revert back to empty string